code/game/gamemodes/miniantags/guardian/guardian.dm

```diff
@@ -80,6 +84,17 @@
 	summoner << "<span class='danger'><B>Your [name] died somehow!</span></B>"
 	summoner.death()
 
+
+/mob/living/simple_animal/hostile/guardian/proc/updatehudhealth()
```

Comment:
use `/mob/living/simple_animal/hostile/guardian/handle_hud_icons_health()` and remove the Life() hook